tests/test-cases/Group23-VIC-Machine-Service/23-02-VCH-List.robot

```diff
@@ -26,21 +26,16 @@ Setup
     Start VIC Machine Server
     Install VIC Appliance To Test Server
 
-
 Teardown
-    Cleanup VIC Appliance On Test Server
```

Comment:
It seems like we'd want this to run if the VCH still exists, to ensure we don't leave anything behind if a failure occurs.

I'm not sure whether the right pattern is to check to see if it exists and then cleanup, or to just attempt to cleanup ignoring any failures.

Reply:
fixed